homeassistant/components/evohome/__init__.py

```diff
@@ -523,6 +522,11 @@ def get_session_id(client_v1: ev1.EvohomeClient) -> str | None:
                 ),
                 err,
             )
+            self.temps = {}  # high-presicion temps now stale
+
+        except Exception:  # pylint: disable=broad-except
+            self.temps = {}  # high-presicion temps now stale
+            raise
```

Comment:
I am not sure if this broad except is acceptable to the HA devs. 

I can certainly remove it.